modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/js/ckeditor4/BaseEditor.js

```diff
@@ -62,6 +63,13 @@ const BaseEditor = forwardRef(
 		}, []);
 
 		useEffect(() => {
+			if (firstRenderRef.current) {
+				firstRenderRef.current = false;
+			}
```

Comment:
Looks good! Only thought, would it make more sense to use our hook `useIsMounted`? It is in frontend-js-react-web

Reply:
cc'ing @markocikos because not sure if this notified you, as I did a comment and not review

Reply:
I do get a GitHub notification on any comment, in `Participating` filter that I am always checking. There is a `Mentioned` filter, but I find `Participating` to have acceptable level of noise (CI comments).

As for `useIsMounted`, I tried it, but I don't think it's applicable in this case. `isMounted()` returns `true` even on first render. IIUC, it's because it's [implemented](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-react-web/src/main/resources/META-INF/resources/js/hooks/useIsMounted.ts#L30) with `useLayoutEffect`, resolving before first render. This I feel is tool for a different kind of problems.